jx.hcl

```diff
@@ -9,4 +9,9 @@ darwin {
   source = "https://github.com/jenkins-x/jx/releases/download/v${version}/jx-darwin-amd64.tar.gz"
 }
 
-version "3.2.137" {}
\ No newline at end of file
+version "3.2.137" "3.2.140" {}
+
+channel "stable" {
```

Comment:
FWIW Hermit generates a `latest` channel that points to the latest stable version. OTOH the package has no control over it, so this may be preferable. Up to you.

Reply:
ah thanks! I tried installing a new version and hermit rejected it - maybe I need to wait 24 hours or something for it to refresh stable?

Reply:
Documented [here](https://cashapp.github.io/hermit/packaging/reference/#channels).

Reply:
`hermit sync` will sync the latest package repos down.

Reply:
I couldn't see from the docs how/when `latest` updates?

Reply:
It's in the section I linked:

> Additionally, Hermit will create several synthetic channels which are checked for updates every **24h**:

Easy to miss, there are a lot of docs :)